src/index.html

```diff
@@ -7,6 +7,421 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img src="./img/logo.png" alt="moyo logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div
+        class="card"
```

Comment:
Would be great to use `cards` for the class name 
```suggestion
    <main class="cards">
      <div
        class="cards__card"
```
![image](https://github.com/mate-academy/layout_catalog/assets/99403198/b304fd3b-24fe-4dd7-90ae-72ce5fde6a70)

Reply:
done